### DISCUSSION TITLE
Is it possible to show/hide tabs depending on a condition?

### DISCUSSION THREAD

**Discussion by christian-reichart** [payloadcms/payload#1664] [Q&A] [answered]
Currently, according to the type definition, a tab should support the `admin` property, but it get's ignored and I couldn't find it in the documentation.

I'm trying to build a table block, that allows the creation of tables.

My idea was the following:
Specify a `numberOfColumns` field that allows numbers between 2 and 10.
Depending on that number, 2-10 tabs would show up below for each "row" in an array of "rows" to allow filling the cells.

Is this possible? Do you have any other ideas how to realise a table block with the given fields or what a custom component would look like to achieve this?

Thanks

**Comment by JessRynkar** [comment, marked as answer]
Hi @christian-reichart conditional tabs is on our roadmap here: https://github.com/payloadcms/payload/discussions/1840
